gytheio-commons/pom.xml

```diff
@@ -4,7 +4,7 @@
     <parent>
         <groupId>org.gytheio</groupId>
         <artifactId>gytheio-parent</artifactId>
-        <version>0.12-SNAPSHOT</version>
+        <version>0.11.3-SNAPSHOT</version>
```

Comment:
We shouldn't go backwards. I suggest either a new branch from SP/0.11.N (and that can then be merged forward to master) *or* simply cherry-pick back to SP/0.11.N after fixing on master.

Reply:
Makes sense, I forgot about the SP branch. Thanks.